### ISSUE TITLE
units of SF_val_max_decomp

### ISSUE THREAD

**Issue by rgknox** [NGEET/fates#533]
The fates parameter "SF_val_max_decomp" reports units of kgC/m2/year in the parameter file:

```	
float fates_max_decomp(fates_litterclass) ;
fates_max_decomp:units = "kgC/m2/yr ?" ;
fates_max_decomp:long_name = "maximum rate of litter & CWD transfer from non-decomposing class into decomposing class" ;
```

However, following through the code, it looks like the units are just standard turnover units of "per year" [year-1].  See example of usage here (indexing soon to be outdated):

https://github.com/NGEET/fates/blob/master/biogeochem/EDPhysiologyMod.F90#L1393

In brief:           flux = mass * sf_val_max_decomp * scaler

[kg/m2/year] = [kg/m2] * [/year] * [-]

I would then propose to:

1) fix the units in code and in the parameter file
2) maybe move this parameter to edparams? (its not used in the fire code... but I suppose its whole purpose is to determine how much litter sticks around to be acted upon by fire...)

Confirmation or recommendations?

@jkshuman @rosiealice

**Comment by rgknox** [comment]
Also, if we move this parameter to edparams, should we also move "fates_CWD_frac" with it?

**Comment by rosiealice** [comment]
Yes, (the units are turnover fraction) and yes, they probably shouldn't be in fire. Can they have their own namespace variable, something like: 

fates_cwd_sizefrac
fates_cwd_turnover

?

**Comment by jkshuman** [comment]
@rgknox it looks like this was addressed in PR #530 
Am I reading that right?

**Comment by rgknox** [comment]
I think so, the units were changed and the name looks better:
https://github.com/NGEET/fates/pull/530/files#diff-a236898dbb7848d5cfdf2610b0ba32c0R448
